incubator/hnc/config/manager/manager.yaml

```diff
@@ -32,8 +32,8 @@ spec:
         resources:
           limits:
             cpu: 100m
-            memory: 30Mi
```

Comment:
While 30MB was too low, 1GB is almost certainly too high. Also, this belongs in its own PR.